### DISCUSSION TITLE
Post processing using pyaedt

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4958] [Q&A] [answered]
Hi, In my model i have created temperature contours which needs to be exported , for which i am using the command export_image, though i am able to export the image it is very small as i have a sufficiently large air region. 
1. Is there any way to hide air region 
2. Can the temperature scale generated be moved using script? or made transparent as it overlaps with the contours.

**Comment by lorenzovecchietti** [comment, marked as answer]
Did you try the `export_image` method? It has the `show_region` argument that you can set to false.
It is not possible to modify the scale, but you can change the resolution of the exported image and that might reduce the dimension of the scale compared to the rest of your model. You can also try to use the `selection` argument of the method so that the view is correctly fitted to your objects. Let me know if these suggestions are helpful.
